.docker/entrypoint/spec/jekyll_server_spec.rb

```diff
@@ -18,36 +18,67 @@
   end
 
   describe '#execute :serve' do
-    describe 'weird file' do
-      data_dir = File.join(__dir__, '..', '..', '..', 'tests', 'full')
-
-      before(:all) do
-        @thread = Thread.new do
-          context = Context.new('development', __dir__, data_dir)
-          jekyll_config_provider = JekyllConfigProvider.new(context)
-          context.configuration = jekyll_config_provider.provide('serve')
-          jekyll_server = JekyllServer.new(context)
-          jekyll_server.execute
-        end
-        @thread.abort_on_exception = true
-
-        JekyllServe.mutex.synchronize do
-          running = JekyllServe.running?
-          JekyllServe.run_cond.wait(JekyllServe.mutex) unless running
-        end
+    data_dir = File.join(__dir__, '..', '..', '..', 'tests', 'minimal')
+    
+    before(:all) do
+      @thread = Thread.new do
+        context = Context.new('development', __dir__, data_dir)
+        jekyll_config_provider = JekyllConfigProvider.new(context)
+        context.configuration = jekyll_config_provider.provide('serve')
+        jekyll_server = JekyllServer.new(context)
+
+        jekyll_server.execute
       end
+      @thread.abort_on_exception = true
 
-      after(:each) do
-        JekyllServe.shutdown
+      JekyllServe.mutex.synchronize do
+        running = JekyllServe.running?
+        JekyllServe.run_cond.wait(JekyllServe.mutex) unless running
+      end
+    end
+
+    after(:all) do
+      JekyllServe.shutdown
+
+      JekyllServe.mutex.synchronize do
+        running = JekyllServe.running?
+        JekyllServe.run_cond.wait(JekyllServe.mutex) if running
+      end
+      FileUtils.remove_dir(File.join(data_dir, '_site') , force = true)
+    end
+
+    describe 'weird file' do
+      weird_filename = '0.0.0.0'
+      it {
+        expect(File).not_to exist(File.join(__dir__, weird_filename))
+      }
+      it {
+        expect(File).not_to exist(File.join(__dir__, '..', weird_filename))
+      }
+      it {
+        expect(File).not_to exist(File.join(data_dir, weird_filename))
+      }
+    end
 
-        JekyllServe.mutex.synchronize do
-          running = JekyllServe.running?
-          JekyllServe.run_cond.wait(JekyllServe.mutex) if running
-        end
+    describe 'site is not empty' do
+      subject do
+        Pathname.new(File.join(data_dir, '_site'))
       end
 
       it {
-        expect(File).not_to exist(File.join(__dir__, '..', '0.0.0.0'))
+        is_expected.to be_directory
+      }
+
+      it {
+        is_expected.to exist
+      }
+
+      it {
+        expect(Dir.empty? subject).to eq(false)
```

Comment:
Codacy found an issue: [Add parentheses to nested method call `Dir.empty? subject`.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5820027)